shared/src/commonTest/kotlin/co/touchlab/kampkit/BreedModelTest.kt

```diff
@@ -42,29 +45,45 @@ class BreedModelTest : BaseTest() {
         settings.putLong(BreedModel.DB_TIMESTAMP_KEY, currentTimeMS)
         assertTrue(ktorApi.mock.getJsonFromApi.calledCount == 0)
 
-        assertNull(model.getBreedsFromNetwork())
+        val expectedError = DataState.Error("Unable to download breed list")
+        val actualError = model.getBreedsFromNetwork(0L)
+
+        assertEquals(
+            expectedError,
+            actualError
+        )
         assertTrue(ktorApi.mock.getJsonFromApi.calledCount == 0)
     }
 
+    @ExperimentalTime
     @Test
     fun updateFavoriteTest() = runTest {
         ktorApi.mock.getJsonFromApi.returns(ktorApi.successResult())
-        assertNull(model.getBreedsFromNetwork())
-        val breedOld = dbHelper.selectAllItems().first().first()
-        assertFalse(breedOld.isFavorited())
+        dbHelper.deleteAll()
+        val appenzeller = Breed(1, "appenzeller", 0L)
+        val australianNoLike = Breed(2, "australian", 0L)
+        val australianLike = Breed(2, "australian", 1L)
 
-        model.updateBreedFavorite(breedOld)
+        val dataStateSuccessNoFavorite = DataState.Success(
+            ItemDataSummary(appenzeller, listOf(appenzeller, australianNoLike))
+        )
 
-        val breedNew = dbHelper.selectById(breedOld.id).first().first()
-        assertEquals(breedNew.id, breedOld.id)
-        assertTrue(breedNew.isFavorited())
+        val dataStateSuccessFavorite = DataState.Success(
+            ItemDataSummary(appenzeller, listOf(appenzeller, australianLike))
+        )
+
+        model.getBreeds().test {
+            assertEquals(DataState.Loading, expectItem())
+            assertEquals(dataStateSuccessNoFavorite, expectItem())
+            model.updateBreedFavorite(australianNoLike)
+            assertEquals(dataStateSuccessFavorite, expectItem())
+        }
```

Comment:
Add an `expectNoEvents()` on the end here to ensure there's nothing else going on that you didn't assert on. It can also be nice to do an `expectNoEvents()` before the `updateBreedFavorite()` line to ensure that the last emission is in response to that call and didn't come earlier.